core/src/test/java/io/questdb/cutlass/line/tcp/NewLineProtocolParserLexerTest.java

```diff
@@ -0,0 +1,212 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2020 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cutlass.line.tcp;
+
+import java.nio.charset.StandardCharsets;
+
+import org.junit.Assert;
+
+import io.questdb.cutlass.line.LineProtoException;
+import io.questdb.cutlass.line.LineProtoLexerTest;
+import io.questdb.cutlass.line.tcp.NewLineProtoParser.ErrorCode;
+import io.questdb.cutlass.line.tcp.NewLineProtoParser.ParseResult;
+import io.questdb.cutlass.line.tcp.NewLineProtoParser.ProtoEntity;
+import io.questdb.std.Chars;
+import io.questdb.std.Numbers;
+import io.questdb.std.Unsafe;
+import io.questdb.std.str.StringSink;
+import io.questdb.test.tools.TestUtils;
+
+public class NewLineProtocolParserLexerTest extends LineProtoLexerTest {
```

Comment:
Where this class is used? I don't see @test(s) defined in it somehow

Reply:
It extends LineProtoLexerTest which has tests in it